src/test/java/io/dashboardhub/pipelinedashboard/definitions/GuestDefinitions.java

```diff
@@ -21,6 +21,5 @@ public void I_try_to_access_a_secure_page() {
     @Then("^I get redirected to the login page$")
     public void I_get_redirected_to_the_login_page() {
         Assert.assertEquals(withBaseUrl("/login"), driver.getCurrentUrl());
-        driver.close();
```

Comment:
Good question, because I share the functionality between **contexts**, it closes it at the end only. This might present an issue in the future, but won't know until then